Tools/RemoteConsole/ly_remote_console/ly_remote_console/remote_console_commands.py

```diff
@@ -171,20 +172,20 @@ def _scan_ports(self):
         max_port = self.port + num_ports_to_scan
         while self.port < max_port:
             try:
+                print(f'trying port {self.addr}:{self.port}')
                 self.socket.connect((self.addr, self.port))
+                print(f'connected {self.addr}:{self.port}')
```

Comment:
use logger.info instead of prints

Reply:
This is to indicate ongoing connection process, it can take time, I believe the user should know what goes on, if it is connected or still tries to connect.